CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# C#/WinRT Contributor's Guide
+
+Below is our guidance for how to report issues, propose new features, and submit contributions via Pull Requests (PRs).
+
+## Before you start, file an issue
+
+Please follow this simple rule to help us eliminate any unnecessary wasted effort & frustration, and ensure an efficient and effective use of everyone's time - yours, ours, and other community members':
+
+> 👉 If you have a question, think you've discovered an issue, would like to propose a new feature, etc., then find/file an issue **BEFORE** starting work to fix/implement it.
+
+### Search existing issues first
+
+Before filing a new issue, search existing open and closed issues first: It is likely someone else has found the problem you're seeing, and someone may be working on or have already contributed a fix!
+
+If no existing item describes your issue/feature, great - please file a new issue:
+
+### File a new Issue
+
+* Don't know whether you're reporting an issue or requesting a feature? File an issue
+* Have a question that you don't see answered in docs, videos, etc.? File an issue
+* Want to know if we're planning on building a particular feature? File an issue
+* Got a great idea for a new utility or feature? File an issue/request/idea
+* Don't understand how to do something? File an issue/Community Guidance Request
+* Found an existing issue that describes yours? Great - upvote and add additional commentary / info / repro-steps / etc.
+
+### Complete the template
+
+**Please include as much information as possible in your issue**. The more information you provide, the more likely your issue/ask will be understood and implemented. Helpful information includes:
+
+* What device you're running (inc. CPU type, memory, disk, etc.)
+* What build of Windows your device is running
+
+  👉 Tip: Run the following in PowerShell Core
+
+  ```powershell
+  C:\> $PSVersionTable.OS
+  Microsoft Windows 10.0.18909
+  ```
+
+  ... or in Windows PowerShell
+
+  ```powershell
+  C:\> $PSVersionTable.BuildVersion
+
+  Major  Minor  Build  Revision
+  -----  -----  -----  --------
+  10     0      18912  1001
+  ```
+
+  ... or Cmd:
+
+  ```cmd
+  C:\> ver
+
+  Microsoft Windows [Version 10.0.18900.1001]
+  ```
+

```

Comment:
We should also ask about dotnet version, cswinrt version, sdk projection version.